content/auth-provider-comparison/index.md

```diff
@@ -0,0 +1,197 @@
+- Pricing tiers for additional MAUs:
+![Cognito pricing tiers](./cognito_pricing_tiers.png)
+- Additional SAML/OIDC users:
+  - **$0.015** per MAU after the first **50** in the free tier
+- Pricing tiers for advanced security features : 
+![Cognito security pricing](./cognito_security_pricing.png)
+
+- SMS charges for multi-factor: 
+  - Inbound charges:
+  ![sms inbound charge](./cognito_sms_pricing_inbound.png)
+  - Outbound changes:
+  ![sms outbound charge](./cognito_sms_pricing_outbound.png)
+
+### So is Cognito the way to go?
+Cognito's free tier and integration with other AWS services position it as a great auth provider, but it does not provide the best developer experience. Users have complained about the documentation being confusing at times, making it hard to integrate with.
+
+## <a name="supertokens" href="">SuperTokens</a>
+![supertokens logo](./supertokens_logo.png)
+
+SuperTokens is an open-source authentication solution that provides a managed service for hassle-free auth and an on-prem solution so developers can manage their data. 
+
+Starting as a session management solution in 2019, SuperTokens has evolved into a complete auth solution providing email-password, social and passwordless login with extensive customization options to curate the login experience to your needs.
+
+SuperTokens differentiates itself from its competitors with its flexibility. If you don't want to deal with handling user data you can use the managed service or, you can self-host the solution for free and control 100% of the data with your database. There are also no restrictions on completely customising the auth flow with their **override** feature. 
+
+
+### Advantages of using SuperTokens:
+#### Completely Open Source:
+- All of SuperTokens SDKs including the core service are open source. This allows developers to poke around the code for vulnerabilities and contribute to features.
+#### Simple Setup:
+- SuperTokens modular architecture makes the setup simple. It allows the developer to pick and choose what functionality they want and only see documentation relevant to their use case.
+- Pre-built Auth UI that can be embedded in your website natively
+- Support over [Discord](https://supertokens.com/discord).
+####  Flexibility: 
+- As mentioned previously Supertokens has a managed service with reasonable pricing as well as a completely free self-hosted version.
+- SuperTokens provides an override feature that enables you to fully customise authentication flows on both your frontend and backend. SuperTokens also provides pre-built UI components that can be rendered directly on your frontend
+
+###  Disadvantages of using SuperTokens:
+#### Enterprise features:
+- SuperTokens currently doesn't offer certain enterprise features like enterprise connections (Active directory,/SAML) or compliance certifications.
```

Comment:
for supertokens disadvantages, remove that we don't support SAML (cause we do via boxyhq). Just say "some enterprise features are missing".

Another disadvantage is that we don't have support for all tech stacks -> missing ruby, php, java, haskell, c++ etc + no pre built UI for angluar / vue